monai/apps/reconstruction/my_mri_array.py

```diff
@@ -0,0 +1,279 @@
+# Copyright (c) MONAI Consortium
```

Comment:
Yes it's just mypy would raise weird errors locally for array.py, I guess it would confuse it with other arrays under `monai.transforms`.
In my latest commit, I named those `mri_array.py` and `mri_dictionary.py`, but as @rijobro suggested, I'll rename them to `array.py` and `dictionary.py` once I move them to `monai.transforms.mri/` after the I address all the reviewers' comments.
Thanks for pointing this out!